src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -409,6 +409,20 @@ PARAM_DEFINE_FLOAT(FW_LND_AIRSPD_SC, 1.3f);
  */
 PARAM_DEFINE_FLOAT(FW_LND_THRTC_SC, 1.0f);
 
+/**
+ * Go around throttle threshold
+ *
+ * The vehicle will go around if throttle position larger than threshold
+ * during approach. it will trigger if throttle position passed from below
+ * to above threshold
+ * @min 0.0
+ * @max 1.0
+ * @decimal 2
+ * @increment 0.1
+ * @group FW L1 Control
+ */
+PARAM_DEFINE_FLOAT(FW_GOTA_TH, 0.9f);
```

Comment:
What does "GOTA" stand for here?  I'm guessing Go Around something something.... it's not very obvious. 
What about "_TH"  is that throttle or threshold?  Either way, "_THR" is more commonly used in the FW L1 Control group.
Maybe use "FW_GOAROUND_THR"?

Reply:
OK